rules/linux/process_creation/proc_creation_lnx_gtfobin_rsync_shell.yml

```diff
@@ -0,0 +1,29 @@
+title: Rsync GTFOBin Abuse - Linux
+id: e2326866-609f-4015-aea9-7ec634e8aa04
+status: experimental
+description: Detects usage of rsync as a GTFOBin Linux binary abuse to break out from restricted environments by spawning an interactive system shell. 
+references:
+    - https://gtfobins.github.io/gtfobins/rsync/#shell
+author: Li Ling, Andy Parkidomo, Robert Rakowski, Blake Hartstein (Bloomberg L.P.)
+date: 2024/08/05
+tags:
+    - attack.execution
+    - attack.t1059
+logsource:
+    category: process_creation
+    product: linux
+detection:
+    selection_img:
+        Image|endswith: '/rsync'
+        CommandLine|contains: ' -e '
+    selection_cli:
+        CommandLine|contains:
+            - 'sh 0<&2 1>&2'
+            - 'bash 0<&2 1>&2'
+            - 'dash 0<&2 1>&2'
```

Comment:
Can not use `sh 1>&2 0<&2` to bypass the rule ?
contains 'sh...' cover 'bash...' and 'dash...' or other *sh like zsh

Reply:
Good point! please review the latest commit for the detection:
    selection_cli:
        CommandLine|contains:
            - 'sh 0<&2 1>&2'
            - 'sh 1>&2 0<&2'
           
This logic will cover the mentioned bypass and *sh